src/main/java/com/thebois/views/WorldView.java

```diff
@@ -14,7 +14,7 @@
 
     private final Color grassColor = new Color(0.005f, 0.196f, 0.107f, 1);
     private ArrayList<ITerrain> terrainTiles = new ArrayList<>();
-    private float tileSize;
+    private final float tileSize;
```

Comment:
Have you tested that it scales correctly after making this final?

Reply:
Yes, since we actually never change the size on the tiles. When we resize the window it scales everything inside the viewport which means the tile size remains the same. And if you check everywhere we use tile size we never actually update it.